Topics/Tech_Stacks/React.md

```diff
@@ -1,37 +1,59 @@
-It's ease of use and popularity comes from how little you actually have to code. Easily applicable for any frontend interface, if you're going to use any library/framework for frontend development, React is the safest option.
+React is an open-source JavaScript library for building component-based user interfaces. This component-based architecture allows engineers to quickly prototype and develop complex applications easily by modularizing common functionalities into discrete, reusable UI **s, and React boasts a large user base full of freely available components libraries. Beyond this, React utilizes a virtual DOM, JSX, React Native, and much more -- so much more that it can be [overwhelming](https://auth0.com/blog/how-to-manage-javascript-fatigue/). 
 
-### React Resources 
+To help ease the learning curve, below you'll find a variety of resources to reference while learning and components and libraries to use with React. 
 
-These frontend challenges for HTML, CSS and JavaScript will help you review core skills while also making interesting projects as well. \
-https://www.frontendmentor.io/
+### React Learning Resources 
 
+[Full-Stack Open](https://fullstackopen.com/)
+A full, free course provided by the University of Helsinki that teaches the MERN (MongoDB, Express, React, NodeJS) stack. It also covers TypeScript, GraphQL, React Native, and some CI/CD. An excellent resource for React beginners.
 
-Basic React tutorial provided by the official React website. This tutorial will guide you through a very basic tutorial that you can do straight from your browser. \
-https://reactjs.org/tutorial/tutorial.html
+[Coursera Learn React Basics](https://www.coursera.org/learn/react-basics)
+A free course provided by Coursera that covers the basics of React, including State, navigation, and taking user input via forms.
 
+[Free Scrimba React Course](https://scrimba.com/learn/learnreact)
+A free course provided by Scrimba that teaches React fundamentals through the creation of various applications including an AirBnB clone, a meme generator, and a notes app.
 
-React tutorial by MDN Web Docs. Useful readings for if you have time, but not necessary by any means.\
-https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/React_getting_started
+[Official React Tutorial](https://reactjs.org/tutorial/tutorial.html)
+Basic React tutorial provided by the official React website. This tutorial will guide you through a very basic tutorial that you can do straight from your browser. 
 
+[React JS Crash Course](https://www.youtube.com/watch?v=w7ejDZ8SWv8)
+[One Hour React Tutorial](https://www.youtube.com/watch?v=b9eMGE7QtTk)
+One hour in-depth tutorials about React. These Youtube crash course provides a great overview of React and various important components that you will be using often. Very useful if you are learning React under a strict time limit for whatever reason.
```

Comment:
"One hour" and "in-depth" are conflicting descriptors, it might make more sense if we don't call it in-depth.